pycm/pycm_curve.py

```diff
@@ -258,6 +341,31 @@ def __curve_thresholds_handler__(curve, thresholds):
         curve.thresholds = thresholds
         if isinstance(curve.thresholds, numpy.ndarray):
             curve.thresholds = curve.thresholds.tolist()
+        curve.thresholds = sorted(curve.thresholds)
+
+
+def __curve_data_filter__(curve):
+    """
+    Filter data.
```

Comment:
It would be better if this description provided more details. My suggestion:
"Eliminate and refine the points at which the curve is undefined."

Reply:
Fixed in https://github.com/sepandhaghighi/pycm/pull/467/commits/270f77c9af42a0cd65ca272bc002c38e374244bd